etc/kayobe/kolla/globals.yml

```diff
@@ -48,14 +48,14 @@ keepalived_tag: 7.0.2.1-1
 keystone_tag: 7.0.2.1-1
 kibana_tag: 7.1.0.1
 kolla_toolbox_tag: 7.0.2.1-1
-logstash_tag: 7.1.0.1
```

Comment:
Confession: I'm not sure how some of the image tags in globals have gone backwards.

Reply:
There should be no need to change this one - we haven't patched the image.

Reply:
Looks good now apart from this

Reply:
Once again you're correct. For a moment I thought I'd put my monasca-log-transform fixes in kolla - they use the logstash image and would require updating the tag. In fact, those fixes are in kolla-ansible.